models/userCartDao.js

```diff
@@ -0,0 +1,76 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+//장바구니 리스트 불러오는 request시 수행할 동작
+const allUserCart = async user_id => {
+  const getAllUserCart = await prisma.$queryRaw`
+    SELECT
+      carts.id, 
+      product_id, 
+      products.kor_name,
+      products.eng_name,
+      products.price,
+      products.sale_rate
+    FROM
+      carts
+    JOIN
+      products
+    ON
+      product_id = products.id
+    WHERE
+      user_id = ${user_id};
+    `;
+  return getAllUserCart;
+};
+
+//장바구니에 상품 추가 request시 수행할 동작
+const addUserCart = async REQUIRED_KEYS => {
```

Comment:
```suggestion
const addUserCart = async (user_id, product_id) => {
```

왜냐하면 Dao는 controller에서 관리하는 required_keys에 대해 의존적이어서는 안되기 때문입니다.